tests/unit/test_database.py

```diff
@@ -62,7 +64,7 @@ class _CredentialsWithScopes(
     return mock.Mock(spec=_CredentialsWithScopes)
 
 
-class _BaseTest(unittest.TestCase):
+class _BaseTest(OpenTelemetryBase):
```

Comment:
Why is this change needed?

Reply:
@harshachinta it was needed in order for me to invoke self.get_finished_spans() and others.